src/main/kotlin/se/uulm/snowballr/backend/service/ProjectService.kt

```diff
@@ -1,26 +1,45 @@
 package se.uulm.snowballr.backend.service
 
+import se.uulm.snowballr.backend.auth.GrpcContext
 import se.uulm.snowballr.backend.db.dummyUserId
 import se.uulm.snowballr.backend.grpc.SnowballRServer.SnowballRService
 import se.uulm.snowballr.backend.model.AccessType
 import se.uulm.snowballr.backend.model.EntityType
 import se.uulm.snowballr.backend.model.SnowballRException.UnauthorizedException
+import se.uulm.snowballr.backend.model.dto.Project
 import se.uulm.snowballr.backend.model.dto.toGrpcProject
 import se.uulm.snowballr.backend.model.parseUUID
 import se.uulm.snowballr.backend.repository.IProjectTableRepo
 import se.uulm.snowballr.backend.repository.IUserTableRepo
-import snowballr.ProjectOuterClass
+import snowballr.Base
+import snowballr.ProjectOuterClass.ProjectStatus
+import snowballr.ProjectOuterClass.Project as GrpcProject
 
 interface IProjectService {
     /**
      * Service implementation of [SnowballRService.createProject].
      */
-    suspend fun createProject(request: ProjectOuterClass.Project.Create): ProjectOuterClass.Project
+    suspend fun createProject(request: GrpcProject.Create): GrpcProject
 
     /**
      * Service implementation of [SnowballRService.getAllProjects].
      */
-    suspend fun getAllProjects(): ProjectOuterClass.Project.List
+    suspend fun getAllProjects(): GrpcProject.List
+
+    /**
+     * Service implementation of [SnowballRService.getAllProjectsForUser]
+     */
+    suspend fun getAllProjectsForUser(request: Base.Id): GrpcProject.List
+
+    /**
+     * Service implementation of [SnowballRService.getAllArchivedProjectsForUser]
+     */
+    suspend fun getAllArchivedProjectsForUser(request: Base.Id): GrpcProject.List
+
+    /**
+     * Service implementation of [SnowballRService.getAllDeletedProjectsForUser]
```

Comment:
None of these service implementations are connected to the server calls.
```suggestion
     * Service implementation of [SnowballRService.getAllProjectsForUser].
     */
    suspend fun getAllProjectsForUser(request: Base.Id): GrpcProject.List

    /**
     * Service implementation of [SnowballRService.getAllArchivedProjectsForUser].
     */
    suspend fun getAllArchivedProjectsForUser(request: Base.Id): GrpcProject.List

    /**
     * Service implementation of [SnowballRService.getAllDeletedProjectsForUser].
```

Reply:
I totally forgot this, thank you.